### ISSUE TITLE
raco fails with "package is already installed"

### ISSUE THREAD

**Issue by ulysses4ever** [Bogdanp/setup-racket-cache-example#1]
Here's our failing job: https://github.com/nuprl/website/runs/4060978881

We have the following incantation in `.github/workflows`:
```
    steps:
      - name: Checkout
        uses: actions/checkout@v2
      - name: Cache Racket dependencies
        uses: actions/cache@v2
        with:
          path: |
            ~/.cache/racket
            ~/.local/share/racket
          key: default
      - name: Install Racket
        uses: Bogdanp/setup-racket@v0.8
        with:
          architecture: 'x64'
          distribution: 'full'
          version: '7.9'
          packages: 'gregor,frog,honu'
```

Is caching supposed to work in the presence of `raco`-packages?

**Comment by Bogdanp** [comment]
I would recommend removing the `packages: ...` line and adding an explicit step to install the dependencies. For example:

```
      ...
      - name: Install Racket
        uses: Bogdanp/setup-racket@v0.8
        with:
          architecture: 'x64'
          distribution: 'full'
          version: '7.9'
      - name: Install Dependencies
        run: raco pkg install --auto --skip-installed gregor frog honu
```

The trick here is to pass the `--skip-installed` flag to `raco pkg install`, which the `packages: ...` syntax doesn't know how to do.

Also, I would recommend upgrading to the latest version of setup-racket (1.6.1).

**Comment by ulysses4ever** [comment]
@Bogdanp thanks for explaining! We can do explicit installation, but here's the question: why not update the action to pass the flag there? The flag can't hurt, right? And succinct and declarative description of package dependencies in the action parameters is preferable to a separate step, I think.

We will upgrade the action version soon, as this is just an oversight on our side. Thanks for noticing!

**Comment by Bogdanp** [comment]
Yes, I think it would probably be fine to add that flag. My one concern would be that I don't know which Racket version it was introduced in, but I think it was probably before any of the versions `setup-racket` supports, so it should be fine.

**Comment by Bogdanp** [comment]
Done! I'm going to close this for now and the change will be released with `setup-racket@v1.7`, which I will release after Racket 7.3 is released (probably this week or the next).

**Comment by ulysses4ever** [comment]
Awesome, thanks a lot @Bogdanp!
